### PR TITLE
Add example for LR scheduler

### PR DESCRIPTION
**Pull request by aweers**
A short and simple example, especially for new users. It helps for a general understanding on how to use this callback. Open for feedback and suggestions. ;)

### CODE REVIEW

File: tensorflow/python/keras/callbacks.py [hunk 1]
```diff
@@ -1168,6 +1168,23 @@ class LearningRateScheduler(Callback):
           (integer, indexed from 0) and returns a new
           learning rate as output (float).
       verbose: int. 0: quiet, 1: update messages.
+  
+  ```python
+  # Firstly, let's create a function which
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: Consider rephrasing to:
```
  # This function keeps the learning rate at 0.001 for the first ten epochs and  
  # decreases it exponentially after that.
```

File: tensorflow/python/keras/callbacks.py [hunk 2]
```diff
@@ -1168,6 +1168,23 @@ class LearningRateScheduler(Callback):
...
+  # maps a given epoch to a learning rate.
+  # This function would keep the learning rate 
+  # constant at 0.001 for the first ten epochs and  
+  # let it decrease exponentially after that.
+  def scheduler(epoch):
+    if epoch < 10:
+      return 0.001
+    else:
+      return 0.001 * tf.math.exp(0.1 * (10 - epoch))
+  # Next, we need to set up the callback and train the model.
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: I think this line of comment may not be needed as the code is self-explanatory.

**Review by rchao — COMMENTED** [review]
Thanks for the PR! Please take a look at the comments.

**Comment by aweers** [comment]
@rchao integrated your suggestions ;)

File: tensorflow/python/keras/callbacks.py (hunk 1 above)
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: Suggesting remove the first sentence "Firstly... to a learning rate."

also increase the length of each line to limit.

Comment (aweers) [reply tensorflow/python/keras/callbacks.py]: Ok, will change line length, but what is the line length limit?

Comment (rchao) [reply tensorflow/python/keras/callbacks.py]: 80 chars per line. Thanks!

**Comment by aweers** [comment]
Resolved issue with line lengths ;)

**Review by rchao — APPROVED** [review]
Thanks! Looks good.
